app/screens/manage_channel_members/manage_channel_members.tsx

```diff
@@ -88,6 +88,7 @@ export default function ManageChannelMembers({
     const [isManageMode, setIsManageMode] = useState(false);
     const [profiles, setProfiles] = useState<UserProfile[]>(EMPTY);
     const hasMoreProfiles = useRef(false);
```

Comment:
I thought you were going to change this initialization to true?

Reply:
My mistake I forgot to do it, it's done now 😄